docs/skopeo-list-tags.1.md

```diff
@@ -72,6 +72,9 @@ This commands refers to repositories using a _transport_`:`_details_ format. The
         "docker.io/myuser/myimage:v1.0"
         "docker.io/myuser/myimage@sha256:f48c4cc192f4c3c6a069cb5cca6d0a9e34d6076ba7c214fd0cc3ca60e0af76bb"
 
+  **docker-archive:**_path_[**:**_docker-reference_]
```

Comment:
Using the `[…docker-reference]` would AFAICS actually not work (it would not be a valid path the `Reader` could consume); so remove that part.